posts/models.py

```diff
@@ -16,3 +17,20 @@ class Post(TimeStampModel):
 
     class Meta:
         db_table = 'posts'
+
+class Photo(models.Model):
+    description = models.TextField()
+    photo       = models.URLField(max_length=300)
```

Comment:
```suggestion
    url       = models.URLField(max_length=500)
```
300은 길이가 너무 짧습니다. 못해도 500이상은 줘야 할 것 같아요!

Reply:
필드명 url로 바꾸고 모두 길이 500으로 수정했습니다!

Reply:
예슬님!! migrate 까지 테스트해보시고 올려주세요! migrations 파일이 하나도 올라오지 않았습니다!

Reply:
헉 완전 잊어버렸네요! migrate 진행하고 다시 올렸습니다!